scss/backend/dot-admin/components/_context-menu.scss

```diff
@@ -0,0 +1,15 @@
+.site-browser__context-menus .site-browser__context-menu {
+    box-shadow: $menu-shadow;
+
+    a {
+        font-size: $font-size-medium;
+        height: $menu-item-height;
+        line-height: $menu-item-line-height;
```

Comment:
I don't see why we need a custom line-height variable, if I'm understanding correctly we should do:

```line-height: $menu-item-height;```